examples/custom/autocomplete/textext_input.coffee

```diff
@@ -0,0 +1,47 @@
+import * as _ from "underscore"
+# Cannot import jquery here, otherwise it overwrites (?) the
+# globally loaded jquery object with function plugins

```

Comment:
Can you [briefly] walk me through this, bearing in mind I have no JS experience? Oddly, if I uncomment the above jquery import, then nothing happens. When I insert a breakpoint in `@render` with the import uncommented, `$.fn.tokenInput === undefined`, and with the import commented, `$.fn.tokenInput !== undefined`.

I'd love to do things the right way here, but this was the culmination of frustration trying to understand the hierarchies/interactions of these modules and imports.